Modules/Scripted/VectorToScalarVolume/VectorToScalarVolume.py

```diff
@@ -76,6 +103,26 @@ def setup(self):
     # Add vertical spacer
     self.layout.addStretch(1)
 
+  def onRGBCheckBox(self, state):
+    if (state == 2): # Checked
+      # Disable other box.
+      self.singleComponentBox.setChecked(False)
+      self.componentsComboBox.setEnabled(False)
+
+  def onSingleComponentCheckBox(self, state):
+    if (state == 2): # Checked
```

Comment:
Parenthesis are now needed in python.

Reply:
For future reference, Qt enum value can be accessed using `qt.Qt.Checked`

Reply:
`qt.Qt.Checked` good to know.